functions/functionsv2/helloworld/hello_world.go

```diff
@@ -0,0 +1,36 @@
+// Copyright 2022 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+// [START functions_helloworld_get]
+
+// Package helloworld provides a set of Cloud Functions samples.
+package helloworld
+
+import (
+	"fmt"
+	"net/http"
+
+	"github.com/GoogleCloudPlatform/functions-framework-go/functions"
+)
+
+func init() {
+	functions.HTTP("HelloGet", helloGet)
+}
+
+// helloGet is an HTTP Cloud Function.
+func helloGet(w http.ResponseWriter, r *http.Request) {
```

Comment:
Should the first `h` be capitalized?

i.e. `HelloGet`?

Reply:
With declarative function signatures, you no longer need to export your function for them to work (the name of the function is based off the first string parameter in the `functions.HTTP()` call) Given that, it's probably best to keep it package local to avoid exposing it to other packages that might try and take a dependency on the function that the author doesn't want.